packages/services/api/src/shared/schema.ts

```diff
@@ -15,10 +15,14 @@ import {
   visit,
 } from 'graphql';
 import lodash from 'lodash';
-import { emptySource, Schema, SchemaObject } from './entities';
-
-export function hashSchema(schema: Schema): string {
-  return createHash('md5').update(schema.source, 'utf-8').digest('hex');
+import type { SchemaObject } from './entities';
+
+export function hashSchema(schema: SchemaObject): string {
+  return createHash('md5')
+    .update(schema.source, 'utf-8')
+    .update(`service_name: ${schema.source}`)
```

Comment:
why are we including service name and url in the hash?

Reply:
https://github.com/kamilkisiela/graphql-hive/blob/8d10cb0d96fb15aa2b6c5680aa9b149527f9c940/packages/services/api/src/modules/schema/providers/schema-validator.ts#L76

Reply:
Because a service name and url can change?